core/src/main/java/com/sun/mail/util/PropUtil.java

```diff
@@ -149,6 +162,20 @@ private static int getInt(Object value, int def) {
 	return def;
     }
 
+	/**
+	 * Interpret the value object as an integer,
+	 * returning def if unable.
+	 */
+	private static ScheduledExecutorService getScheduledExecutorService(Object value) {
```

Comment:
Copy pasted API docs.  Maybe just delete it?

Reply:
done, deleted